packages/core/src/reducer.ts

```diff
@@ -23,13 +24,19 @@ export default combineReducers({
   dictateInterims,
   dictateState,
   language,
-  lastTypingAt,
   notifications,
   readyState,
   referenceGrammarID,
   sendBoxValue,
   sendTimeout,
   sendTypingIndicator,
   shouldSpeakIncomingActivity,
-  suggestedActions
+  suggestedActions,
+  typing,
+
+  // TODO: [P3] Take this deprecation code out when releasing on or after 2022-02-16
+  lastTypingAt,
+
+  // TODO: [P3] Take this deprecation code out when releasing on or after January 13 2020
+  sendTyping: sendTypingIndicator
```

Comment:
I think we have a PR open to remove this

Reply:
merged in - should be resolved with rebase